ros2_batch_job/__main__.py

```diff
@@ -516,6 +515,21 @@ def run(args, build_function, blacklisted_package_names=None):
         job.run(['"%s"' % job.python, '-m', 'pip', '--version'], shell=True)
         # Install pip dependencies
         pip_packages = list(pip_dependencies)
+
+        # Mypy is a special case.  We prefer to get it from the distribution, if it exists there.
+        # If not, we want to install it via pip.  So we try to import it here to see if it exists.
+        need_mypy_from_pip = False
+        try:
+            import mypy
+        except ModuleNotFoundError:
+            need_mypy_from_pip = True
+
+        if need_mypy_from_pip:
```

Comment:
Just for the rhythm of it :grin:

```suggestion
        need_mypy_from_pipy = False
        try:
            import mypy
        except ModuleNotFoundError:
            need_mypy_from_pipy = True

        if need_mypy_from_pipy:
```